worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -66,10 +66,18 @@
     @NotNull
     @Size(min = 1)
     private String jobDatabasePassword;
+    
+    /**
+     * The application name when connecting to the Job Database.
+     */
+    @NotNull
+    @Size(min = 1)
+    private String appName;
 
 
     public JobTrackingWorkerReporter() throws JobReportingException {
         this.jobDatabaseURL = Objects.requireNonNull(JobDatabaseProperties.getDatabaseUrl()).toLowerCase(Locale.ENGLISH);
+		this.appName = Objects.requireNonNull(JobDatabaseProperties.getApplicationName());
```

Comment:
- Alignment
- App Name should be optional.  Otherwise this would be a breaking change.